src/gdcdictionary/schemas/other_clinical_attribute.yaml

```diff
@@ -0,0 +1,1261 @@
+        - Hyperthyroidism
+        - Hypospadias
+        - Hypothyroidism
+        - Hypothyroidism
+        - Inflammatory Bowel Disease
+        - Insulin Controlled Diabetes
+        - Interstitial Pneumonitis or ARDS
+        - Intraductal Papillary Mucinous Neoplasm
+        - Iron Overload
+        - Ischemic Heart Disease
+        - ITP
+        - Joint Replacement
+        - Kidney Disease
+        - Li-Fraumeni Syndrome
+        - Liver Cirrhosis (Liver Disease)
+        - Liver Toxicity (Non-Infectious)
+        - Low Grade Liver Dysplastic Nodule
+        - Lupus
+        - Lymphamatoid Papulosis
+        - Lymphocytic Meningitis
+        - Lymphocytic Thyroiditis
+        - Lymphoid Hyperplasia
+        - Lynch Syndrome
+        - MAI
+        - Malaria
+        - Metabolic Syndrome
+        - Methicillin-Resistant Staphylococcus aureus (MRSA)
+        - Myasthenia Gravis
+        - Mycobacterium avium Complex
+        - Myocardial Infarction
+        - Neuroendocrine Tumor
+        - Nodular Hyperplasia
+        - Nonalcoholic Steatohepatitis
+        - Obesity
+        - Organ Transplant (Site)
+        - Osteoarthritis
+        - Osteoporosis or Osteopenia
+        - Other Cancer within 5 Years
+        - Other Nonmalignant Systemic Disease
+        - Other Pulmonary Complications
+        - Pain (Various)
+        - Pancreatitis
+        - Peptic Ulcer (Ulcer)
+        - Peripheral Neuropathy
+        - Peripheral Vascular Disease
+        - Peutz-Jeghers Disease
+        - Pneumocystis Pneumonia
+        - Pneumonia, NOS
+        - Polycystic Ovarian Syndrome (PCOS)
+        - Pregnancy in Patient or Partner
+        - Primary Sclerosing Cholangitis
+        - Psoriasis
+        - Pulmonary Fibrosis
+        - Pulmonary Hemorrhage
+        - Renal Dialysis
+        - Renal Failure (Requiring Dialysis)
+        - Renal Insufficiency
+        - Rheumatoid Arthritis
+        - Rheumatologic Disease
+        - Rubinstein-Taybi Syndrome
+        - Sarcoidosis
+        - Seizure
+        - Shingles
+        - Sjogren's Syndrome
+        - Sleep Apnea
+        - Smoking
+        - Staph Osteomyelitis
+        - Staphylococcus aureus
+        - Steatosis
+        - Stroke
+        - Syphilis
+        - Thyroid Disease, Non-Cancer
+        - Transient Ischemic Attack
+        - Treponema pallidum
+        - Tuberculosis
+        - Turcot Syndrome
+        - Tyrosinemia
+        - Ulcerative Colitis
+        - Urinary Tract Infection
+        - Varicella Zoster Virus
+        - Wagr Syndrome
+        - Other
+        - Unknown
+        - Not Reported
+    enumDef:
+      Cancer:
+        $ref:
+          - "_terms_enum.yaml#/cancer/common"
+      Epstein-Barr Virus:
+        $ref:
+          - "_terms_enum.yaml#/epsteinbarr_virus/common"
+      Pneumocystis Pneumonia:
+        $ref:
+          - "_terms_enum.yaml#/pneumonia_nos/common"
+      Wagr Syndrome:
+        $ref:
+          - "_terms_enum.yaml#/other/common"
```

Comment:
the ref seems a little weird to me for Wagr Syndrome

Reply:
Yeah ref got added for wrong enum (Should be for Other), will fix it